buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -266,14 +266,6 @@ class ClusterFormationTasks {
                 environment esEnv
                 errorOutput = new ByteArrayOutputStream()
```

Comment:
I would remove this line, so that if there is some error output, it will be printed to the console.